new/PRS14178Zadengel.xml

```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14178Zadengel" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Zadəngəl</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-11-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘድንግል</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zadəngəl</persName>
+                    <persName xml:lang="gez" type="normalized">ʾabbā</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">gərā getā</occupation>
```

Comment:
I understand the text as follows:
"We have ordered it in the following way (አዘዝነ፡ ከመዝ፡ እንዘ፡)
the ṣərāǧ māsre (shall be) ʾabbā Takla Wald,
the qes ʾaḍe (shall be) ʾabbā Giyorgis,
...
the gərā getā of Kəbrān (shall be) ʾabbā Zadəngəl, ..."

If this interpretation is correct, each of these terms in this list should be occupations within the monastery or church of Kəbrān. If there is another interpretation of this addition, it might be neccessary to reconsider also the other records.

Reply:
OK, good